configurator/src/Configurator/Vendor/Keycloak/KeycloakManager.php

```diff
@@ -25,11 +25,18 @@ public function __construct(
 
     private function getAuthenticatedClient(): HttpClientInterface
     {
+        $verifySsl = false;
+        if (getenv('VERIFY_SSL') == 'true') {
+            $verifySsl = true;
+        }
+
         return $this->keycloakClient
             ->withOptions([
                 'headers' => [
                     'Authorization' => 'Bearer '.$this->getToken(),
                 ],
+                'verify_peer' => $verifySsl,
+                'verify_host' => $verifySsl,
             ]);
```

Comment:
This should be included on the http_client config.
And it's already defined for `verify_peer`. Please introduce another env var `VERIFY_HOST`, because it's a different option!